Master-Game/World/TransitionTile.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MasterGame.World
+{
+    class TransitionTile: BaseTile
+    {
+        private bool IsOpenForTransition;
+
+        public TransitionTile(): base(TileType.Transition,true,0.0f)
+        {            
+        }
+
+        /// <summary>
+        /// This method will set the attribute for if the transition tile is open (i.e., = true)
+        /// </summary>
+        /// <param name="newCondition"></param>
+        public void SetIsOpenForTransition(bool newCondition)
+        {
+            IsOpenForTransition = newCondition;
+        }
+
+        public bool GetIsOpenForTransition()
```

Comment:
no needs to call it GetIsOpenForTransition. IsOpenForTransition should be fine

Reply:
See comment above regarding get-set.